pkg/services/ngalert/provisioning/notification_policies.go

```diff
@@ -121,10 +136,14 @@ func (nps *NotificationPolicyService) ResetPolicyTree(ctx context.Context, orgID
 	})
 
 	if err != nil {
-		return definitions.Route{}, nil
+		return definitions.Route{}, "", nil
 	} // TODO should be error?
 
-	return *route, nil
+	version, err := calculateRouteFingerprint(*route)
```

Comment:
Q: I wonder if returning a version is necessary for reset? I don't see it used in the route API draft and it seems that returning an error after the transaction has already succeeded because we can't calculate the fingerprint isn't optimal. Can be avoided by removing the error from the signature, see other comment.

Reply:
originally, it was used in provisioning API and returned as the operation result. I rewrote the code since then, this is artifact from previous version. I will remove it!